PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -117,12 +117,15 @@
 )
 
 # Only add low pT electrons for run2_miniAOD_UL or bParking era
-from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
 from Configuration.Eras.Modifier_bParking_cff import bParking
+from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
 _lowPt_extraCommands = ['keep *_slimmedLowPtElectrons_*_*',
                         'keep *_gsfTracksOpenConversions_*_*',]
-(bParking | run2_miniAOD_UL).toModify(MicroEventContent,
-                                      outputCommands = MicroEventContent.outputCommands + _lowPt_extraCommands)
+(bParking | run2_miniAOD_UL).toModify(
```

Comment:
can the changes in this file be reverted completely?
a simple `git checkout CMSSW_10_6_25 -- PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py`
and then commit could work

Reply:
Changed as suggested.

Reply:
Done.